components/script/dom/webidls/HTMLMediaElement.webidl

```diff
@@ -16,7 +16,7 @@ interface HTMLMediaElement : HTMLElement {
   [CEReactions] attribute USVString src;
   attribute MediaProvider? srcObject;
   readonly attribute USVString currentSrc;
-  // [CEReactions] attribute DOMString crossOrigin;
+  [CEReactions] attribute DOMString crossOrigin;
```

Comment:
This should be `crossOrigin?` per https://html.spec.whatwg.org/multipage/media.html#media-elements:dom-media-crossorigin.